### ISSUE TITLE
Format docstrings

### ISSUE THREAD

**Issue by sloria** [asottile/blacken-docs#32]
It would be grand if this could format code after `::` or `.. code-block:: python` within Python docstrings.

**Comment by asottile** [comment]
oddly, it seems to already work -- just we don't have it opted in against python files:

```console
$ cp t.py t2.py
$ blacken-docs t.py 
t.py: Rewriting...
```

```diff
$ diff -u t.py t2.py
--- t.py	2020-01-14 14:06:44.526632547 -0800
+++ t2.py	2020-01-14 14:06:41.154762613 -0800
@@ -3,5 +3,5 @@
 
     .. code-block:: python
 
-        f(1, 2, 3)
+        f(1,2,3)
     """
```

**Comment by asottile** [comment]
I guess we could expand the files matched in `.pre-commit-config.yaml`? I'm trying to think of cases where the regex matches stuff it shouldn't and can't really think of any -- maybe it's fine?  what do you think @sloria

**Comment by sloria** [comment]
Seems right. Since any Python file can contain a docstring, seems appropriate to match on all .py files.

**Comment by asottile** [comment]
this is available in v1.5.0-1 (tag only due to no code changes)
